packages/web-components/src/components/leadspace/leadspace.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { customElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import DDSLeadSpaceBase from './leadspace-body';
```

Comment:
Let's not forget about changing the file name here.

Reply:
whoops, thanks for catching that!